### DISCUSSION TITLE
NPM package rename to `@keep-network/tbtc` version 2.x

### DISCUSSION THREAD

**Discussion by nkuba** [keep-network/tbtc-v2#97] [Ideas]
What do you think about updating the name of NPM package to the same as we had for V1: `@keep-network/tbtc`, the diference will be with `version` property where we would start with `2.x`
`solidity/package.json` would contain:
```json
  "name": "@keep-network/tbtc",
  "version": "2.0.0-dev",
```

**Comment by michalinacienciala** [comment]
Couple of initial thoughts:
1. Doing as you propose, would mean having something like this in the package.json for tbtc-v2:
```
{
  "name": "@keep-network/tbtc",
  "version": "2.0.0-dev",
[...]
  "dependencies": {
    "@keep-network/tbtc": ">1.1.2-dev <1.1.2-ropsten",
```
Do you know if having the same project (in a different version) as a dependency is something supported by npm?

2. tBTC dashboard currently has a dependency to both `@keep-network/tbtc` and `@keep-network/tbtc-v2`. If we were to do the change you propose, we would have to change it to this (unless there's some other way of handling it I didn't think of):
```
{
  "name": "dashboard",
  "version": "1.17.0-pre",
[...]
  "dependencies": {
    "@keep-network/tbtc": ">1.1.2-dev <1.1.2-ropsten",
    "@keep-network/tbtc": ">2.0.0-dev <2.0.0-ropsten",
```
I don't think such config is accepted.

**Reply by nkuba** [reply]
Nice catch!

We could try aliasing the dependencies, but I'm not sure it will work fine for all our use-cases.
```
"dependencies": {
  "@keep-network/tbtc-v1": "npm:@keep-network/tbtc@^1.1.2-dev",
  "@keep-network/tbtc": ">2.0.0-dev <2.0.0-ropsten",
}
```
